containerd-shim-spin/src/engine.rs

```diff
@@ -45,6 +45,7 @@ impl Default for SpinEngine {
         // turned on for the components we compile.
         let mut config = wasmtime::Config::default();
         config.epoch_interruption(true);
+        config.native_unwind_info(false);
```

Comment:
**suggest**: add a comment explaining why this is here

Reply:
good call out. Added